changelog/20548.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
```

Comment:
@VioletHynes could you please update this changelog file with the format for announcing a new feature? https://hashicorp.atlassian.net/wiki/spaces/VAULT/pages/1311244491/Changelog+Process#New-and-Major-Features

Reply:
Will do! Good catch, sorry for missing this!